pkgs/tools/system/swaysettings/default.nix

```
@@ -0,0 +1,75 @@
+{
+  pkgs,
+  meson,
+  vala,
+  pkg-config,
+  gettext,
+  ninja,
+  python3Full,
+  wrapGAppsHook,
+  appstream-glib,
+  desktop-file-utils,
+  gsettings-desktop-schemas,
+  glib,
+  gtk3,
+  libhandy,
+  json-glib,
+  libgee,
+  pantheon,
+  libxml2,
+  accountsservice,
+  gobject-introspection,
+  dbus
+}:
+with pkgs;
+pkgs.stdenv.mkDerivation rec {
+  pname = "swaysettings";
+  version = "0.0";
+  src = pkgs.fetchFromGitHub {
+    owner = "ErikReider";
+    repo = "SwaySettings";
+    rev = "master";
+    hash = "sha256-EVGY4LbiH+RB56sYGOkYCdOc01LqbJIg9mJjgvFd4oY=";
+  };
+
+  nativeBuildInputs = [
+                        meson
+                        vala
+                        pkg-config
+                        gettext
+                        ninja
+                        python3Full
+                        wrapGAppsHook
+                        appstream-glib
+                        desktop-file-utils
+                        ];
+  buildInputs = [
+                  gsettings-desktop-schemas
+                  glib
+                  gtk3
+                  libhandy
+                  json-glib
+                  libgee
+                  pantheon.granite
+                  libxml2
+                  accountsservice
+                  gobject-introspection
+                  dbus
+                  ];
+
+  meta = with pkgs.lib; {
```

```suggestion
  meta = with lib; {
```

At least one year ago `lib` was split from `stdenv`

---

Corrected